src/packages/@ncigdc/modern_components/SampleType/SampleType.js

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import { compose } from 'recompose';
+
+import { withTheme } from '@ncigdc/theme';
+import withRouter from '@ncigdc/utils/withRouter';
+
+export default compose(withRouter, withTheme)(({ repository }) => {
+  return (
+    <td>
```

Comment:
Don't need this td here, it'll end up double wrapped in a td: 
<img width="201" alt="screen shot 2017-11-20 at 4 37 18 pm" src="https://user-images.githubusercontent.com/1314446/33042792-17b42348-ce11-11e7-8f29-e184b892cf3e.png">
return span like you were before, or directly return string because react 16 lets ya do that https://reactjs.org/blog/2017/09/26/react-v16.0.html#new-render-return-types-fragments-and-strings 🎉

Reply:
oops yeah i was messing around with the elements because of the huge spaces. turns out all i had to do was remove the minHeight in relay